packages/design-system/src/components/chip/Chip.tsx

```diff
@@ -0,0 +1,28 @@
+import type { PropsWithChildren } from "react";
+
+import * as S from "./chip.css";
+
+interface ChipProps {
+  variant: "contained" | "outlined";
+  icon?: React.ReactNode;
+}
+
+export const Chip = ({
+  variant,
+  children,
+  icon,
+}: PropsWithChildren<ChipProps>) => {
+  return (
+    <>
+      <span
+        className={[
+          S.chipContainer,
+          variant === "contained" ? S.contained : S.outlined,
+        ].join(" ")}
+      >
+        {icon}
+        {children}
+      </span>
+    </>
```

Comment:
```suggestion
      <span
        className={[
          S.chipContainer,
          variant === "contained" ? S.contained : S.outlined,
        ].join(" ")}
      >
        {icon}
        {children}
      </span>
```
최상위에 빈 태그로 안감싸줘도 될 것같아요!

Reply:
이거 chip컴포넌트 내용이였네요...! main에서 적용해서 수정해놓을게요!